<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVoltageTable</class>
 <widget class="QDialog" name="DialogVoltageTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voltage table</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="logo_label">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/logo_tripleReg.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <widget class="QPushButton" name="cancel_pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Voltage table</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>368</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>PMT anode voltage</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="anodeFrom_spinBox">
        <property name="suffix">
         <string> V</string>
        </property>
        <property name="maximum">
         <number>8000</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="anodeTo_spinBox">
        <property name="suffix">
         <string> V</string>
        </property>
        <property name="minimum">
         <number>2000</number>
        </property>
        <property name="maximum">
         <number>8000</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="anodeStep_spinBox">
        <property name="suffix">
         <string> V</string>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>8000</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="anode_tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>10</number>
        </attribute>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Anode (V)</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearAnode_pushButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>PMT focusing voltage</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="focusingFrom_spinBox">
        <property name="suffix">
         <string> V</string>
        </property>
        <property name="maximum">
         <number>8000</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="focusingTo_spinBox">
        <property name="suffix">
         <string> V</string>
        </property>
        <property name="minimum">
         <number>660</number>
        </property>
        <property name="maximum">
         <number>8000</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="focusingStep_spinBox">
        <property name="suffix">
         <string> V</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>8000</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="focusing_tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>10</number>
        </attribute>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Anode (V)</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearFocusing_pushButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>368</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="set_pushButton">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>set_pushButton</tabstop>
  <tabstop>cancel_pushButton</tabstop>
  <tabstop>anodeFrom_spinBox</tabstop>
  <tabstop>anodeTo_spinBox</tabstop>
  <tabstop>anodeStep_spinBox</tabstop>
  <tabstop>anode_tableWidget</tabstop>
  <tabstop>clearAnode_pushButton</tabstop>
  <tabstop>focusingFrom_spinBox</tabstop>
  <tabstop>focusingTo_spinBox</tabstop>
  <tabstop>focusingStep_spinBox</tabstop>
  <tabstop>focusing_tableWidget</tabstop>
  <tabstop>clearFocusing_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
